service-python/pdfgenerator/src/lib/templates/shared/intro.html

```diff
@@ -81,9 +81,9 @@ <h1>VHA {{document_type|capitalize}} Data Summary</h1>
   <i>Generated automatically on {{timestamp.strftime('%m/%d/%Y')}} at {{timestamp.strftime('%H:%M %p %Z')}}</i>
   <br /><br />
   
-  <p class="m-0">{{veteran_info.first}} {{veteran_info.middle}} {{veteran_info.last}} {{veteran_info.suffix}}</p>
+  <h1 class="m-0" style="font-size: 21px; margin-top: 24px;">{{veteran_info.first}} {{veteran_info.middle}} {{veteran_info.last}} {{veteran_info.suffix}}</h1>
 
-  <p>DOB: {{veteran_info.birthdate}}</p>
+  <p>DOB: {{veteran_info.birthdate.strftime('%m/%d/%Y')}}</p>
```

Comment:
Waiting on response from Shannon [here](https://dsva.slack.com/archives/C01Q7979Z7D/p1668628435307339?thread_ts=1668617826.587689&cid=C01Q7979Z7D)

Reply:
DOB format is good (for now)! No change needed. Please merge, including to the `main` branch being used for end-to-end testing.